projects/admin-core/src/lib/shared/services/admin-auth.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Router } from '@angular/router';
+import { interval, switchMap, take } from 'rxjs';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { DestroyRef, Inject, Injectable } from '@angular/core';
+import { TAILORMAP_SECURITY_API_V1_SERVICE, TailormapSecurityApiV1ServiceModel } from '@tailormap-viewer/api';
+import { ConfirmDialogService } from '@tailormap-viewer/shared';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AdminAuthService {
+
+  private isAuthenticated = false;
+
+  constructor(
+    @Inject(TAILORMAP_SECURITY_API_V1_SERVICE) private api: TailormapSecurityApiV1ServiceModel,
+    private destroyRef: DestroyRef,
+    private dialogService: ConfirmDialogService,
+    private router: Router,
+  ) {
+  }
+
+  public pingUserLoggedIn() {
+    interval(1000 * 50)
+      .pipe(takeUntilDestroyed(this.destroyRef), switchMap(() => this.api.getUser$()))
```

Comment:
so this will keep calling the API (and keep the session open) as long as a browser tab is active